script/tool/lib/src/lint_android_command.dart

```diff
@@ -0,0 +1,56 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:file/file.dart';
+import 'package:flutter_plugin_tools/src/common/plugin_utils.dart';
+import 'package:platform/platform.dart';
+
+import 'common/package_looping_command.dart';
+import 'common/process_runner.dart';
+
+const String _gradleWrapper = 'gradlew';
```

Comment:
If the idea is that it should eventually run on Windows, then it would also need to consider that this is gradlew.bat on Windows.

Reply:
I've been doing Windows support on an as-needed basis so far, but there's no reason I couldn't add it to this one out of the box. It would be useful for contributors.